frontend/test/__support__/metadata.ts

```diff
@@ -0,0 +1,12 @@
+import { getMetadata, MetadataSelectorOpts } from "metabase/selectors/metadata";
+import { createMockState } from "metabase-types/store/mocks";
+import { createEntitiesState, EntitiesStateOpts } from "./store";
+
+export function createMockMetadata(
+  entities: EntitiesStateOpts,
+  opts?: MetadataSelectorOpts,
+) {
+  const state = createMockState({ entities: createEntitiesState(entities) });
+  const metadata = getMetadata(state, opts);
+  return { metadata, state };
+}
```

Comment:
Not yet sure if giving back state is a good idea, but looking at the `DashCardMenu` test it feels handy. Looking forward for more feedback

Reply:
Would it make sense to add a shorthand for creating metadata for the sample database preset? 🤔